### DISCUSSION TITLE
DEenrichRPlot error

### DISCUSSION THREAD

**Discussion by sghoshuc** [satijalab/seurat#7058] [Q&A]
Hi Is there any tutorial for DEenrichRPlot. there is nothing under vignettes. 
I am getting following error when running DEenrichRPlot.
Must specify at least one of url or handle
Error in do.call(what = cbind, args = pos.er) : 
  second argument must be a list

I am running the following code
 DEenrichRPlot(merged_IU_filt_rD,
                       ident.1 = "1_CON",
                       ident.2 = "1_CR",
                       balanced = TRUE,
                       logfc.threshold = 0.25,
                       assay = NULL,
                       max.genes=50,
                       test.use = "wilcox",
                       enrich.database = "GO_Biological_Process_2013",
                       p.val.cutoff = 0.05)

Thanks in advance.

**Comment by maminby** [comment]
try loading package enrichr by run `library(enrichR)`
